apps/fluent-tester/src/FluentTester/TestComponents/ButtonExperimental/ButtonSizeTestSection.tsx

```diff
@@ -48,15 +48,24 @@ export const ButtonSizeTest: React.FunctionComponent = () => {
       <Button loading size="large" style={commonTestStyles.vmargin}>
         Loading Button Large
       </Button>
-      <CompoundButton secondaryContent="rounded" shape="rounded" style={commonTestStyles.vmargin}>
+      <CompoundButton secondaryContent="Small compound button" style={commonTestStyles.vmargin}>
```

Comment:
I would pass in a size explicitly (so set size="small"). The default size is different on each platform, so on a different OS this will show as a medium button